array.rb

```diff
@@ -1,59 +1,60 @@
 class Array
   # call-seq:
-  #    ary.shuffle!              -> ary
-  #    ary.shuffle!(random: rng) -> ary
+  #    array.shuffle! -> array
```

Comment:
Not sure if we need this line, since the next also covers the case where no argument is provided.  Similar issue in a couple methods below.

Reply:
Fixed.